os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1351,12 +1242,11 @@ void up_serialinit(void)
 #endif
 
 #ifdef CONFIG_PM
-	/* Register to receive power management callbacks */
-	ret = pm_register(&g_serialpm.pm_cb);
+	/* Domain registration for operation control */
+	(void)pm_domain_register("UART");
+	(void)pm_domain_register("LOGUART");
```

Comment:
Is it necessary to distinguish between UART and LOGUART?

pm_domain manages the count of pm_suspend in domain units.
This is to ensure that the use of pm by other domains does not affect it.

If the uart and loguart processes the same logic, "UART" is enough.

Reply:
Hi Eunwoo, I agree with you. LOGUART will be removed.